resources/manage-user

```diff
@@ -1,71 +1,13 @@
 #!/usr/bin/env python3
+#
+# Manage users for repository access through basic authentication.
+#
 
-"""Manage users for repository access through basic authentication."""
+import sys
+from os import path
 
-from pathlib import Path
-import subprocess
-import argparse
+sys.path.append(path.join(path.dirname(__file__), 'lib'))
 
+from archive_auth_mirror.scripts import manage_user
 
-AUTH_FILE = Path(__file__).parent.parent.absolute() / 'basic-auth'
```

Comment:
The only real change is that we don't need to declare AUTH_FILE here, since we can use get_paths()